### PR TITLE
Add cli command to clean up local data store

### PR DESCRIPTION
**Pull request by vachanw**
Addresses: https://github.com/Netflix/metaflow/issues/41

Simplistic implementation (for now) to clean up and reset your local data store. Adds a CLI command to achieve this. When invoked this will look for and delete the data store file. 

Screen Shot 2019-12-12 at 13 27 20

Screen Shot 2019-12-12 at 13 22 58

### CODE REVIEW

**Comment by romain-intel** [comment]
Assigning to Ravi for review and merging.

**Comment by seeravikiran** [comment]
Could you please move this to a named branch (say 'fix-issue-41') so I can do some testing before I can leave comments.
We have put quite a lot of time into the polish on the CLI UX, so I definitely have comments to ensure the indentation and the color scheme are in check.

**Comment by savingoyal** [comment]
@seeravikiran External contributors can't create branches in this repo (for obvious reasons). You can directly clone from the commit hash - `bcae819d6e54c74cb5961f4f4e95fa8eecaf1602`

File: metaflow/main_cli.py [hunk 1]
```diff
@@ -123,6 +124,21 @@ def status():
...
+    
+    if path:
+        if click.confirm("This will reset your local data store." +\
+                        click.style("\nWARNING:", fg='red') +\
+                                    "This step cannot be undone!\nProceed to delete: " +\
+                                    click.style("\"{0}\"".format(path), fg='red') + "?"):
+            shutil.rmtree(path)
+    else:
+        # Nothing to do here since we didnt find an existing store.
+        pass
```
Comment (seeravikiran) [inline metaflow/main_cli.py]: Please display a relevant message to the user even if there was nothing to do. I think it is very surprising once you execute a command and see nothing happen.

See `configure reset` if you want ideas on how we handle it for configuration.

**Comment by mpkocher** [comment]
Perhaps it would be useful to consider `rm -rf /path/to-datastore` as a reasonable solution for deleting a local datastore. The layer of indirection in this PR might not be worth the complexity it's adding.

**Comment by vachanw** [comment]
alright, if we want to leave this capability as `rm -rf /path/to-datastore`, I can close the PR. @savingoyal @seeravikiran

**Comment by vachanw** [comment]
I think more broadly, we should disallow creation of the datastore in arbitrary locations. Perhaps default it to `~/.metaflow/local/datastore` or something.  This may be easy to add to the platform, what do you folks think?

**Comment by savingoyal** [comment]
@romain-intel Would you like to weigh in on this?

**Comment by romain-intel** [comment]
This is definitely an additional level of indirection but it is a minor change that may make things easier. I'll check with @seeravikiran if he is OK with the new flow (with confirmation and what not) and we can either merge this or leave it as rm -rf as others have suggested.

**Comment by seeravikiran** [comment]
I vote for closing this PR and just allowing users to use the `rm -rf /path/to/datastore`. Sorry for the delayed response.

**Comment by romain-intel** [comment]
Closing since the consensus seems to just do rm -rf.

**Pull request closed without merging**